imm.yaml

```diff
@@ -59,23 +59,20 @@
     Id: da4ed776-0365-4d8a-a297-c4e91a14d646
     Measures:
         -
-            Score: 0
-            Description: None
-        -
+            Date: 2020-03-16
 -
     Name: 'Framework Version'
     Id: 6c0402b3-f0e3-4bd7-83fe-04bb6dca7924
     Measures:
         -
-            Score: 3
-            Description: 'Using the most current LTS version'
+            Framework: netcoreapp3.1
```

Comment:
Should this have a score?

Reply:
No, I modified the IMM badge code to calculate the score based on its list of which frameworks are current, so that the score drops automatically any time a framework moves out of LTS, or a new version becomes the latest LTS version.